### PR TITLE
add rocketmq grayscale UT test

### PR DESCRIPTION
**Pull request by chengyouling**
What type of PR is this?

UT tests

What this PR does / why we need it?

Function coverage test for mq grayscale.

Which issue(s) this PR fixes？

#1539

Does this PR introduce a user-facing change?

No

### CODE REVIEW

File: sermant-plugins/sermant-mq-grayscale/mq-config-common/src/test/java/io/sermant/mq/grayscale/config/MqGrayConfigCacheTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+        // serviceMeta changed will not update config
+        GrayTagItem item = CommonConfigUtils.getGrayTagItem();
+        item.getServiceMeta().put("x_lane_tag", "red" );
+        MqGrayscaleConfig config1 = getMqGrayscaleConfig(item);
+        MqGrayConfigCache.setCacheConfig(config1, DynamicConfigEventType.MODIFY);
+        Assert.assertEquals("gray",
+                MqGrayConfigCache.getCacheConfig().getGrayscale().get(0).getServiceMeta().get("x_lane_tag"));
+
+        GrayTagItem item1 = CommonConfigUtils.getGrayTagItem();
```
Comment (daizhenyu) [inline sermant-plugins/sermant-mq-grayscale/mq-config-common/src/test/java/io/sermant/mq/grayscale/config/MqGrayConfigCacheTest.java:54]: comment out the function of each test case.

File: sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/service/RocketMqGraySendMessageHookTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * RocketMqGraySendMessageHook test
+ *
+ * @author chengyouling
+ * @since 2024-09-10
+ **/
+public class RocketMqGraySendMessageHookTest extends RocketMqTestAbstract {
+    @Test
+    public void testSchedulerCheckGrayConsumerStart() {
+        Message message = new Message();
```
Comment (daizhenyu) [inline sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/service/RocketMqGraySendMessageHookTest.java:38]: Test the situation that the same label exists.

File: sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/RocketMqTestAbstract.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+/**
+ * test abstract
+ *
+ * @author chengyouling
+ * @since 2024-09-10
+ **/
+public class RocketMqTestAbstract {
+    private MockedStatic<ConfigManager> configManagerMockedStatic;
+
+    @Before
```
Comment (daizhenyu) [inline sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/RocketMqTestAbstract.java]: Do you need to execute the before method before each test? . If not, use beforeclass.

Comment (chengyouling) [reply sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/RocketMqTestAbstract.java]: fixed

File: sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/service/RocketMqGraySendMessageHookTest.java (hunk 2 above)
Comment (chengyouling) [reply sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/test/java/io/sermant/mq/grayscale/rocketmq/service/RocketMqGraySendMessageHookTest.java:38]: fixed

File: sermant-plugins/sermant-mq-grayscale/mq-config-common/src/test/java/io/sermant/mq/grayscale/config/MqGrayConfigCacheTest.java (hunk 1 above)
Comment (chengyouling) [reply sermant-plugins/sermant-mq-grayscale/mq-config-common/src/test/java/io/sermant/mq/grayscale/config/MqGrayConfigCacheTest.java:54]: fixed

**Review by lilai23 — DISMISSED** [review]

**Review by daizhenyu — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Pull request merged**